docs/Interoperator-SM-authentication.md

```diff
@@ -0,0 +1,22 @@
+# Service Fabrik Inter-operator Authentication
```

Comment:
This looks like a redundant document since the content is similar to docs/Interoperator-authentication.md